src/components/Navigation.js

```diff
@@ -3,7 +3,8 @@ import React from 'react';
 
 const Navigation = () => (
   <nav>
-    <ul>
+    <ul className="navbar">
+      <li><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSmoDV73D3hV_4yUErBD746LWfh_cru5spf5g&usqp=CAU" alt="house" /></li>
       <li><a href="#m">House</a></li>
       <li><a href="#d">Reserveform</a></li>
       <li><a href="#c">MyReservations</a></li>
```

Comment:
The result should look exactly the same (with exception for titles and images) as in [the design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) provided by [Murat Korkmaz](https://www.behance.net/muratk) on Behance. 

- We must use a sidebar panel instead of a navbar. that's a requirement . 
- Use the same colors from the provided design.